bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/XToPsiNSG/VBFToH_HToJPsiG_JPsiToMuMu/VBFToH_HToJPsiG_JPsiToMuMu_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+set default_unset_couplings 99
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model jpsi_sm
+define p = u c d s u~ c~ d~ s~
+generate p p > h p p $$ z / a p g h w+ w-, (h > jpsi a, jpsi > mu+ mu-)
+add process p p > h p p $$ w+ w- / a p g h z, (h > jpsi a, jpsi > mu+ mu\
```

Comment:
Why do you need to define 2 process lines like this instead of simply `p p > h j j $$ z w+ w-, (h > jpsi a, jpsi > mu+ mu-)` ?

Reply:
It was just for clarity, but it is debatable. I am ok to use your suggestion. I will first try and check if there are any issues